Neos.ContentRepositoryRegistry/Classes/Command/SubprocessProjectionCatchUpCommandController.php

```diff
@@ -21,6 +21,11 @@ class SubprocessProjectionCatchUpCommandController extends CommandController
      */
     protected $contentRepositoryRegistry;
 
+    /**
+     * @param string $contentRepositoryIdentifier
+     * @param string $projectionClassName
```

Comment:
```suggestion
     * @param string $projectionClassName fully qualified class name of the projection to catch up
```

It's the FQN in this case (the command is internal though)